controls/SV-230332.rb

```diff
@@ -87,7 +87,7 @@
     !input('central_account_management')
   }
 
-  if os.release.to_f >= 8.2
+  if Gem::Version.new(os.release) >= Gem::Version.new('8.2')
```

Comment:
We never call direct ruby in controls as it executes on the runner not the target. 

The `round() method off Float would likely be the better solution.

Something like `os.release.to_f.rount(2)` may be what you are looking for. Chaining it this way off an inspec resource ensures the code runs on the target - in either local or remote scanning.